pkg/kv/kvserver/allocator_scorer.go

```diff
@@ -70,6 +70,30 @@ const (
 	// consider a store full/empty if it's at least minRebalanceThreshold
 	// away from the mean.
 	minRangeRebalanceThreshold = 2
+
+	// maxL0SublevelThreshold: if the number of l0 sublevels of a store descriptor
+	// is greater than this value, it will never be used as a rebalance target.
+	maxL0SublevelThreshold = 20
```

Comment:
I think it says 10 on the PR release note, just a heads up to make sure the numbers match at the end.

Reply:
I believe this is 20 in the release note, I've updated this to be clearer and reworked the language.